djangocms_link/templates/djangocms_link/default/link.html

```diff
@@ -14,4 +14,4 @@
         {{ instance.phone }}
     {{ instance.target }}
     {{ instance.attributes_str }}
-{% endcomment %}
\ No newline at end of file
+{% endcomment %}
```

Comment:
it doesn't seem like it https://github.com/divio/djangocms-link/pull/100/commits/8efaf29527d474024b5fb4d6dc4057a7cbd478f9 

(or this is a bug in github)